screens/resources/ResourcesScreen.js

```diff
@@ -45,13 +48,12 @@ class ResourcesScreen extends React.Component {
   render() {
     return (
       <View>
-        <NavButton onPress={() => this.props.navigation.goBack(null)}>
-          <FontAwesome5 name="arrow-left" solid size={24} />
-        </NavButton>
-
-        <View style={{ height: 106 }}>
-          <TopText>Resources</TopText>
-        </View>
+        <NavHeaderContainer>
+          <NavButton onPress={() => this.props.navigation.goBack(null)}>
+            <FontAwesome5 name="arrow-left" solid size={24} />
+          </NavButton>
+          <NavTitle>Resources</NavTitle>
+        </NavHeaderContainer>
```

Comment:
Looks fine on iPhone 11 but on iPhone 8 it the header/"Resources" isn't center-aligned. There is also a gap at the top of the header (also in Points History) that presents itself in < iPhone X versions - not sure if this is a problem/deviation from Figma but just flagging.
![image](https://user-images.githubusercontent.com/35573990/77872964-24abbc80-727b-11ea-956e-cdf85af344d5.png)

Reply:
Good catch! Both of these issues were later fixed in #56